src/app/plugins/quote.plugin.ts

```diff
@@ -0,0 +1,99 @@
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IHttpResponse, IMessage } from '../../common/types';
+import { MessageEmbed } from 'discord.js';
+
+export default class QuotePlugin extends Plugin {
+  public commandName: string = 'quote';
+  public name: string = 'Quote Plugin';
+  public description: string = 'Prints a random quote';
+  public usage: string = 'quote <tag (optional) | list (optional)>';

```

Comment:
Usage supports multiple lines for different examples now, separated by newlines
`quote\nquote <tag>\n quote list`